modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFRequest.path

```diff
@@ -35,6 +35,11 @@
 	<td>//*[contains(@class,'lexicon-icon-plus')]//ancestor::button[contains(text(),'${key_buttonText}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CONTINUE_DISABLED</td>
+	<td>//button[contains(.,'Continue') and @disabled]</td>
+	<td></td>
+</tr>
```

Comment:
Hey @MGabriellaS, this path isn't necessary. We have a generic button file, called `Button.path`, where you can search for this common paths. In this case, you can use `Button#ANY_DISABLED`.

Reply:
Ok,I didn't found before this path. I will remove this one and use Button#ANY_DISABLED